mlflow_databricks_artifacts/azure/client.py

```diff
@@ -0,0 +1,82 @@
+from six.moves import urllib
```

Comment:
This utility is a dependency-free replacement for Azure storage uploads, removing our reliance on the `azure-storage-blob` library. Removal of this dependency ensures that we can embed this library into preexisting software environments without causing unintended package updates.